modules/installation-vsphere-infrastructure.adoc

```diff
@@ -20,6 +20,11 @@
 
 You must install the {product-title} cluster on a VMware vSphere version 6 or 7 instance that meets the requirements for the components that you use.
 
+[IMPORTANT]
+====
+Installing a cluster on VMware vSphere version 6.5 and virtual hardware version 13 is now deprecated. Support for these versions will be removed in a future version of {product-title}.
+====
+
```

Comment:
I feel like we should not deprecate HW version 13 until *we* start defaulting to HW 15 in our images. We are literally deprecating HW 13 *without* providing alternate support for a higher version in OCP-4.9. This seems like bad user experience.

Reply:
There is a ongoing discussion to default to HW version 15 in OCP-4.9 in which case we could deprecate support for HW 13.